src/terminologySectionPanel/ValueSets/Search/SelectSearchCategories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import MultipleSelectDropDown from "./MultipleSelectDropDown";
+
+const SelectSearchCategories = ({
+  formikFieldProps,
+  handleChange,
+  options,
+}) => {
+  return (
+    <div>
+      <MultipleSelectDropDown
+        formControl={formikFieldProps}
+        {...formikFieldProps}
+        id="search-by-category"
+        label="Search By Category"
+        required={false}
+        disabled={false}
```

Comment:
It is better if we can disable fields using canEdit

Reply:
This page is behind a can edit one level up. 

TerminologySectionPanelNavTabs has the tab disabled if user cannot edit. 
It should not be possible to reach this code without canEdit